shared/components/FormField/FormField.jsx

```diff
@@ -167,7 +166,7 @@ FormField.propTypes = {
   feedback: PropTypes.oneOf(['success', 'error']),
   error: PropTypes.string,
   helper: PropTypes.oneOfType([PropTypes.func, PropTypes.node]),
-  tooltip: childrenOfType(Tooltip),
+  tooltip: componentWithName('Tooltip'),
```

Comment:
Since we are changing `FormField`, it is a good idea to give it a package.json and add it as a devDependency to all the components that use it. We did this with `shared/components/Choice`.

This way, any changes to FormField will cause the dependent packages to be re-published.